doc/deimos.rst

```diff
@@ -76,3 +75,13 @@ Fluxing
 If you use the LVMslitC (common), avoid placing your standard
 star in the right-most slit as you are likely to collide with
 a bad column.
+
+Additional Reading
+==================
+
+Here are additional docs related to Keck/DEIMOS:
+
+.. toctree::
+   :maxdepth: 1
+
+   dev/deimosframes
```

Comment:
This is fine for now, but I think anything that's relevant to users in these development reports should get repeated in the main documentation.